recipe/meta.yaml

```diff
@@ -24,11 +24,13 @@ requirements:
     - ninja
     - meson
     - pkg-config
-    - glib  # [build_platform != target_platform]
+    - patch           # [osx]
   host:
     - gobject-introspection
     - glib
     - zlib
```

Comment:
Does it not require `gio` too? See https://github.com/GNOME/gcab/blob/ad0baea50359c1978a9224ee60bf98d97bfb991f/meson.build#L97 and https://docs.gtk.org/gio/index.html.   I'm not familiar with meson, if it is not found dependency it won't generate an error?

Reply:
no, it is not required AFAICS

Reply:
gio is included in glib